### PR TITLE
[Issue 607]Validation for column group

### PR DESCRIPTION
**Pull request by ashokblend**
1.Measure should not be part of column group
2.No dictionary column should not be part of column group
3.same columname should not be specified two different column group
4.complex dimension should not be part of column group
6.timestamp should not be part of column group
7.Added test cases

### CODE REVIEW

**Comment by ashokblend** [comment]
retest this please

File: integration/spark/src/main/scala/org/apache/spark/sql/CarbonSqlParser.scala [hunk 1]
```diff
@@ -517,6 +522,42 @@ class CarbonSqlParser()
...
+    colGrpFieldIndx = colGrpFieldIndx.sorted
+    // check if columns in column group is in schema order
+    if (!checkIfInSequence(colGrpFieldIndx)) {
+      throw new MalformedCarbonCommandException("Invalid column group:" + colGroup)
+    }
+    def checkIfInSequence(colGrpFieldIndx: Seq[Int]): Boolean = {
+      for (i <- 0 until (colGrpFieldIndx.length - 1)) {
+        if ((colGrpFieldIndx(i + 1) - colGrpFieldIndx(i)) != 1) {
+          throw new MalformedCarbonCommandException(
+            "Invalid column group,column group column should be in sequence.")
```
Comment (Vimal-Das) [inline integration/spark/src/main/scala/org/apache/spark/sql/CarbonSqlParser.scala]: This error is confusing, You need the columns to be contiguous as per the schema, not just in sequence.

**Comment by Vimal-Das** [comment]
retest this please

**Comment by Vimal-Das** [comment]
LGTM, Merged to master.

**Pull request merged**